frontend/controller/actions/group.js

```diff
@@ -47,6 +48,14 @@ export async function leaveAllChatRooms (groupContractID: string, member: string
   }
 }
 
+async function saveLoginState (action: string, contractID: string) {
+  try {
+    await sbp('gi.actions/identity/saveOurLoginState')
```

Comment:
Why not handle such errors inside `saveOurLoginState`?

Reply:
Then we wouldn't be able to detect if it failed and log the appropriate message based on what we were doing..